internal/command/bind.go

```diff
@@ -196,112 +112,3 @@ func NewBindCommand(p *KameletPluginParams) *cobra.Command {
 	cmd.Flag("output").Usage = fmt.Sprintf("Output format. One of: %s.", strings.Join(append(printFlags.AllowedFormats(), "url"), "|"))
 	return cmd
 }
-
-func nameFor(name, source string, sinkRef corev1.ObjectReference) string {
-	if name != "" {
-		return name
-	}
-
-	generated := fmt.Sprintf("%s-to-%s-%s", source, sinkRef.Kind, sinkRef.Name)
-
-	generated = filepath.Base(generated)
-	generated = strings.Split(generated, ".")[0]
-	generated = strings.ToLower(generated)
-	generated = disallowedChars.ReplaceAllString(generated, "")
-	generated = strings.TrimFunc(generated, isDisallowedStartEndChar)
-
-	return generated
```

Comment:
what would you do when you bind the same kamelet to the same sink, twice ? E.g. for receiving two different Twitter searches on the same broker ? (or any binding with different configuration, e.g. monitoring to telegram accounts and feeding them to the same broker)

I can't see a logic that tries to detect an existing binding with the calculated name and then potentially add an incrementing suffix (`-2`, `-3`, ...)

Reply:
true, in this case you will hit an error saying that the resource with that name is already present and you will have to use `--name` to set an explicit name for the binding

Reply:
We can catch the error and increment the suffix or find a detection logic for duplicate names

Reply:
Had another closer look at the implementation. When the binding with given name already exists the command will perform an update on the binding. This is how `kamel` uses the bind command. WDYT?

Reply:
I think for `bind` this is ok, but not for `binding create` where you explicitly request a new binding (`binding update` would be for updating a binding), however afais the code is shared amongst both commands.

Reply:
I have added the `--force` option and by default `bind` uses it to auto update existing bindings. `binding create` command will raise an error by default when binding already exists unless `--force` is given